yarn.lock

```diff
@@ -1910,6 +1910,20 @@
   dependencies:
     bowser "^2.5.4"
 
+"@metamask/onboarding@^0.2.1":
+  version "0.2.1"
```

Comment:
Oh, could to dedupe the above version of this as well, and remove it from the manifest? It's no longer used anywhere.